logrus.go

```diff
@@ -6,69 +6,85 @@ import (
 	"os"
 	"time"
 
-	"go.mozilla.org/mozlog"
-
 	"github.com/sirupsen/logrus"
 )
 
-// Enable enables mozlogrus
-func Enable(loggerName string) {
-	logrus.SetFormatter(&MozLogFormatter{
-		LoggerName: loggerName,
-	})
+var hostname string
+var pid int
 
-	logrus.SetOutput(os.Stdout)
-}
+func init() {
+	var err error
+	hostname, err = os.Hostname()
+	if err != nil {
+		hostname = "unknown"
```

Comment:
Failing at this stage would be very odd, and I wonder if we'd want to silently continue like this, rather than fail loudly.

Reply:
maybe instead of "unknown" we use "error-resolving". If you see a bunch of those in the it'll at least make it easy to search. :)